lib/siwapp/invoices.ex

```diff
@@ -5,23 +5,26 @@ defmodule Siwapp.Invoices do
 
   import Ecto.Query, warn: false
   alias Siwapp.Repo
-
   alias Siwapp.Schema.Invoices
 
   @doc """
-  Gets a list of invoices by updated date.
+  Gets a list of invoices by updated date
```

Comment:
This file shouldn't be part of your PR, I don't know what you did but this is not your code